website/docs/examples/targets.md

```diff
@@ -5,6 +5,7 @@ azlinux3/rpm                     Builds an rpm and src.rpm.
 azlinux3/rpm/debug/buildroot     Outputs an rpm buildroot suitable for passing to rpmbuild.
 azlinux3/rpm/debug/sources       Outputs all the sources specified in the spec file in the format given to rpmbuild.
 azlinux3/rpm/debug/spec          Outputs the generated RPM spec file
+azlinux3/testing/sysext          Builds a systemd system extension for testing purposes only.
```

Comment:
FWIW, I think this can go into something like `<worker>/sysext/erofs`
Later I think we could likely come up with an artifact type that can easily pushed to an OCI registry directly and output that at `<worker>/sysext`

Reply:
Given this is all up in the air, having this in `testing/sysext` I guess makes sense for now.